apps/storybook/src/ComboBox.stories.tsx

```diff
@@ -575,3 +575,26 @@ Virtualized.args = {
 Virtualized.argTypes = {
   value: { control: { disable: true } },
 };
+
+export const MultipleSelect: Story<Partial<ComboBoxProps<string>>> = (args) => {
+  const options = React.useMemo(() => countriesList, []);
+  const [selectedOptions, setSelectedOptions] = React.useState<string[]>([]);
+
+  return (
+    <ComboBox
+      options={options}
+      inputProps={{ placeholder: 'Select a country' }}
+      onChange={(value: string, event: string) => {
+        action(value + ' ' + event ?? '')();
+        setSelectedOptions((prev) =>
+          event === 'removed'
+            ? prev.filter((item) => item !== value)
+            : [...prev, value],
```

Comment:
I just realized it's not very nice that the user has to do all this logic in `onChange`. Should we handle it for them and provide the whole array as an argument?